### PR TITLE
bump antrea to 1.7.1

### PR DESCRIPTION
**Pull request by hangyan**
## Which issue(s) this PR fixes

Fixes: #

### CODE REVIEW

File: addons/packages/antrea/1.7.1/bundle/config/kapp-config.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  resourceMatchers:
+  - kindNamespaceNameMatcher: {kind: ConfigMap, namespace: kube-system, name: antrea-ca}
+
+- path: [spec, caBundle]
+  type: copy
+  sources: [existing, new]
+  resourceMatchers:
+  - anyMatcher:
+      matchers:
+      - kindNamespaceNameMatcher: {kind: APIService, namespace: kube-system, name: v1alpha1.stats.antrea.io}
```
Comment (knabben) [inline addons/packages/antrea/1.7.1/bundle/config/kapp-config.yaml:16]: please review these apiservices to see if there are some deprecated or not

Comment (hangyan) [reply addons/packages/antrea/1.7.1/bundle/config/kapp-config.yaml:16]: https://github.com/antrea-io/antrea/blob/v1.7.1/docs/api.md According to this, we have the correct list.

File: addons/packages/antrea/1.7.1/test/Makefile [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Copyright 2021 VMware Tanzu Community Edition contributors. All Rights Reserved.
```
Comment (knabben) [inline addons/packages/antrea/1.7.1/test/Makefile:1]: please make sure the headers are updated correctly

**Comment by hangyan** [comment]
Yeah sure

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Review by knabben — APPROVED** [review]

**Pull request merged**